testng-core-api/src/main/java/org/testng/internal/RuntimeBehavior.java

```diff
@@ -115,6 +116,11 @@ public static boolean isDryRun() {
     return Boolean.parseBoolean(value);
   }
 
+  public static boolean ignoreMissedTestNames() {
```

Comment:
I dont think this is required since we are providing a way to set this via configurations. This means that a user should be able to leverage build tools plugins such as surefire plugin and then set this value.  Please have this removed.

Reply:
this method is used in other places.

Reply:
And this method is to get the System property, not set.

Reply:
I understood now. However, in our case, the values of test names are dynamic and the better way is to use the Vm argument to specify them.

Reply:
@wenijinew - I think the command line arguments should work without having the need for JVM arguments no? We are trying to move away from having to introduce so many JVM arguments and would like to stick to command line arguments that can be set either via `java org.testng.TestNG` (or) via a build tool. I dont think we need a redundant JVM argument way of doing it.

Reply:
OK, I understand. It's not required but better to have to some degree. So, I will remove VM arg in new commit.